web-app/react-app/reducers/integrationKey.js

```diff
@@ -29,6 +30,7 @@ const initialState = {
 export default function(state: IntegrationKeyState = initialState, action: IntegrationKeyAction): IntegrationKeyState {
 
     switch (action.type) {
+        case CREATE_IDENTITY_REQUEST:
```

Comment:
I really think that the reducer files should correspond the action files. So then the actions considering the identity creation should be in `reducers/identity.js`

Reply:
Reducers and actions now reside in integrationkey.js